pyart/aux_io/rainbow_wrl.py

```diff
@@ -384,7 +384,7 @@ def _get_angle(ray_info, angle_step=None, scan_type='ppi'):
     else:
         if angle_step is None:
             raise ValueError('Unknown angle step')
-        angle_start = np.array(ray_info['data']*bin_to_deg, dtype='float64')
+        angle_start = np.array(ray_info[0]['data']*bin_to_deg, dtype='float64')
```

Comment:
The `else` path should only be taken when the `ray_info` is just a dict (consisting of blobid, depth, rays, refid).

The check `if (len(ray_info) == 2):` (Line 375) works only if the `ray_info` contains only `startangle` and `stopangle`.

So, we should check if `ray_info` is a list first. If not we can take the `else` path. If it is a list we iterate over the elements and check `rayinfo[iterator]['@refid']` for it's contents (eg `startangle`, `stopangle`). Then we fill the found data into the right arrays. Makes sense?